.circleci/config.yml

```diff
@@ -18,6 +18,30 @@ parameters:
     type: string
     default: 14.17-browsers
 
+executors:
+  integration-tests:
+    docker:
+      - image: 'cimg/node:<<parameters.node_tag>>'
+      - image: 'circleci/redis:<<parameters.redis_tag>>'
+      - image: 'localstack/localstack:0.11.2'
+        environment:
+          SERVICES: sqs
+          DATA_DIR: /tmp/localstack/data
+          TMP_DIR: /private
+          DEFAULT_REGION: eu-west-2
+    working_directory: ~/app
+    resource_class: <<parameters.resource_class>>
+    parameters:
+      resource_class:
+        default: medium
+        type: string
+      node_tag:
+        default: "14.16-browsers"
+        type: string
+      redis_tag:
+        default: "buster"
+        type: string
+
```

Comment:
Does seem like an orb candidate, as prisoner-offender-search, probation-offender-search, pathfinder, IMS and hmpps-communication-monitoring will all do this. I would have raised to dps-tech-team, but they've all moved into other teams now, so the ORB is "group-maintained". Ok for now, I'll raise in appropriate channel.